docker/docker-compose.yaml

```diff
@@ -12,22 +12,7 @@ services:
       MONGO_INITDB_ROOT_USERNAME: root
       MONGO_INITDB_ROOT_PASSWORD: example
```

Comment:
secrets in source code in open source, really?

Reply:
I haven't found a best practice

Reply:
enviroment variable